modules/genomic_browser/ajax/genomic_file_upload.php

```diff
@@ -120,15 +166,15 @@ function moveFileToFS(&$fileToUpload)
 
     $config           = NDB_Config::singleton();
     $genomic_data_dir = $config->getSetting('GenomicDataPath');
-    $DB =& Database::singleton();
-    reportProgress(40, "Copying file to $genomic_data_dir ");
+
+    reportProgress(98, "Copying file to $genomic_data_dir ");
     if (move_uploaded_file(
         $fileToUpload->tmp_name,
-        $genomic_data_dir . 'genomic_uploader/'
-        . $fileToUpload->file_name
+        $fileToUpload->full_path
     )) {
-        reportProgress(60, "File copied to $genomic_data_dir ");
+        reportProgress(99, "File copied to $genomic_data_dir ");
     } else {
+        endWithFailure();
```

Comment:
Upon failure of `move_uploaded_file`, the progress bar is not updated and hangs after line 170, displaying "Copying file to $genomic_data_dir "